src/main/sensors/rangefinder.c

```diff
@@ -188,7 +188,7 @@ timeDelta_t rangefinderUpdate(void)
 /**
  * Get the last distance measured by the sonar in centimeters. When the ground is too far away, RANGEFINDER_OUT_OF_RANGE is returned.
  */
-int32_t rangefinderRead(void)
+void rangefinderRead(void)
```

Comment:
Let's call it `rangefinderUpdate()` and let's remove `rangefinderCalculateAltitude()`. 
Update function should do all the updating and calculation. Raw and compensated altitude should be accessible by reading rangefinder variables (similar to gyro and acc)